api/src/main/java/org/eclipse/microprofile/config/spi/package-info.java

```diff
@@ -19,31 +19,16 @@
  *******************************************************************************/
 
 /**
- * <p>This package contains classes which are used to extend the standard functionality in a portable way.
- * <p>A user can provide own {@link org.eclipse.microprofile.config.spi.ConfigSource ConfigSources} and
- * {@link org.eclipse.microprofile.config.spi.Converter Converters} to extend the information available in the Config.
+ * This package contains classes which are used to implement the configuration API, and to extend the standard
+ * configuration functionality in a portable way.
+ * <p>
+ * Users and frameworks may provide custom {@link org.eclipse.microprofile.config.spi.ConfigSource} and
+ * {@link org.eclipse.microprofile.config.spi.Converter} instances.  Configuration instances may be set up and created
+ * using the {@link org.eclipse.microprofile.config.spi.ConfigBuilder} API.
+ * <p>
+ * The package also contains the class {@link org.eclipse.microprofile.config.spi.ConfigProviderResolver},
+ * which is used to implement the specification itself.
  *
- * <p>The package also contains the class {@link org.eclipse.microprofile.config.spi.ConfigProviderResolver}
- * which is used to pick up the actual implementation.
- * <h2>Usage</h2>
- * <p>This is used to build up a builder and manually add {@link org.eclipse.microprofile.config.spi.ConfigSource ConfigSources}..</p>
- *
- *
- * <ol>
```

Comment:
I think the info below is useful. Can you put this in the spec under the 4th bulletin of this [section](https://download.eclipse.org/microprofile/microprofile-config-1.3/microprofile-config-spec.html#configprovider)

Reply:
Let's open a new PR for this.